### DISCUSSION TITLE
Use some auto formatting tool for cmake files

### DISCUSSION THREAD

**Discussion by vmiheer** [microsoft/vcpkg#17872] [Ideas]
This is on same lines on why project use `clang-format`, `autopep8` etc.
There are pre-existing tools for cmake formatting:
https://cmake-format.readthedocs.io/en/latest/cmake-format.html
This is related to comment: https://github.com/microsoft/vcpkg/pull/15885#issuecomment-768767285

**Comment by JackBoosY** [comment]
I think what you are referring to has nothing to do with related issues, these formatting tools cannot automatically generate these records.

**Comment by vmiheer** [comment]
My comment: https://github.com/microsoft/vcpkg/pull/15885#discussion_r564375770
https://github.com/microsoft/vcpkg/pull/15885#issuecomment-768767285 is reply to my comment.
